content/posts/blog/2021-01-20.md

```diff
@@ -0,0 +1,55 @@
+---
+title: It is not a choice of the messenger
+date: 2021-01-20 06:30
+author: Edward Maurer
+blog_id: blog
+category: misc
+---
+
+## It is not a choice of the messenger
+
+Recently, several people reached out to me asking what kind of messenger they should be using now. Furthermore, they said that they actually do not understand what is going on and whether they should switch from one of the commonly known messengers to another one. 
+I was wondering how to answer this. Obviously, I could have stood-up and just advocated for XMPP (Extensible Messaging and Presence Protocol), but then I thought this alone is not a helpful answer. On the one hand, I see that many people just make a short-term decision. On the other hand, I also see they usually do not switch because they made a very evaluated choice for one specific messenger.
+
+Actually, most will just add another messenger to their list of messengers. Likely, that is more frustrating than helping, which brings me back to the beginning and the actual question: What issue or what problem does one want to solve here? What are the incentives? Can one decide towards a solution that, when the next hype chases through the internet, will have a better basis?
+
+A possible background of that question may be different expectations of some functionality, the basic requirement of effective data privacy and sovereignty or simply the possibility to reach all my contacts. However, that change will still require leaving behind the previous messaging system and switching to a new one, which almost always results in the loss of the existing contact connections. Furthermore, in terms of privacy, one is almost always required to provide a phone number. Again, what does one will try to solve?
```

Comment:
I'm also not sure what the "background" of a question means but "A possible background of that question" sounds wrong to me. My apologies for not being able to suggest a good replacement here.

I know what you mean by "contact connections" but the phrasing feels awkward. Maybe just "existing contacts"?

It also feels awkward again to ask what question we're trying to answer at the end. Isn't this paragraph explaining the problem we're trying to solve?